src/models/ObjectType.ts

```diff
@@ -1,6 +1,7 @@
 import { ChainObject } from "./ChainObject";
 
 export class ObjectType {
+    public static Unknown = new ObjectType(0, 0);
```

Comment:
Please add TODO to refactor this code, now we have duplicate logic of indexing here.